### PR TITLE
Fix/responsive sidebar

### PR DESCRIPTION
**Pull request by chimdie**

### CODE REVIEW

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]
fix the sidebar for bigcreen also
Screenshot from 2023-06-25 16-02-51

**Comment by chimdie** [comment]
What do I need to change on the big screen? @copy-ninja1

**Comment by chimdie** [comment]
> fix the sidebar for bigcreen also Screenshot from 2023-06-25 16-02-51

Sidebar for big screen fixed
Screenshot 2023-06-25 at 4 23 44 PM

File: layouts/adminLayout.tsx [hunk 1]
```diff
@@ -15,13 +17,41 @@ const AdminLayout: NextPage<InferProps<typeof AdminLayoutProps>> = ({
...
+  const [focused, setFocused] = useState<string | null>(null);
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
+    if (window.innerWidth > 640 && window.innerWidth < 840) {
+      setNavSwitch(true);
+    }
+  }, [navSwitch, setNavSwitch]);
+
+  const handleNavSwitch: () => void = (): void => {
```
Comment (copy-ninja1) [inline layouts/adminLayout.tsx:30]: why returning a function

File: layouts/adminLayout.tsx [hunk 2]
```diff
@@ -15,13 +17,41 @@ const AdminLayout: NextPage<InferProps<typeof AdminLayoutProps>> = ({
   children,
 }) => {
   const router = useRouter();
+  const [navSwitch, setNavSwitch] = useState(false);
```
Comment (copy-ninja1) [inline layouts/adminLayout.tsx:20]: add proper type
```suggestion
  const [navSwitch, setNavSwitch] = useState<boolean>(false);
```

File: layouts/adminLayout.tsx (hunk 1 above)
Comment (copy-ninja1) [reply layouts/adminLayout.tsx:30]:
```suggestion
  const handleNavSwitch = (): void => {
  }
```

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]

File: layouts/adminLayout.tsx (hunk 1 above)
Comment (copy-ninja1) [reply layouts/adminLayout.tsx]: please fix this accordingly

Comment (copy-ninja1) [reply layouts/adminLayout.tsx]: there is no point for the function to return another function

Comment (chimdie) [reply layouts/adminLayout.tsx]: Fixed it

**Review by copy-ninja1 — APPROVED** [review]

**Pull request merged**